app/api/v1/admin/visits.rb

```diff
@@ -11,9 +10,12 @@ def visit
 
       resources :visits do
         desc 'Return all visits'
+        params do
+          optional :sort, type: Hash
+        end
         get do
-          visits = Visit.all
-          present visits
+          default_sort = { column_name: 'id', type: 'asc' }
```

Comment:
it's better to declare the default setting inside of a responsible module, instead of adding it to all controllers in the app